examples/language_identification_example.py

```diff
@@ -0,0 +1,79 @@
+"""Copyright 2022 REV
```

Comment:
Can you make the GitHub Action for updating the copyright years work for this file as well?

Currently, it works on the root level LICENSE